src/web/mjs/connectors/Pixiv.mjs

```diff
@@ -31,29 +41,37 @@ export default class Pixiv extends Connector {
 
     async _getChapters(manga) {
         let chapterList = [];
-        for (let page = 1, run = true; run; page++) {
-            const chapters = await this._getChaptersFromPage(manga.id, page);
-            chapters.length > 0 ? chapterList.push(...chapters) : run = false;
+        if (manga.id.startsWith('artwork-')) {
+            chapterList.push({
+                id: manga.id.match(/artwork-(\d+)/)[1],
+                title: manga.title
+            });
+        } else {
+            for (let page = 1, run = true; run; page++) {
+                const chapters = await this._getChaptersFromPage(manga.id, page);
+                chapters.length > 0 ? chapterList.push(...chapters) : run = false;
+            }
         }
-        return chapterList.filter((chapter, index) => {
-            return index === chapterList.findIndex(item => chapter.id === item.id);
```

Comment:
I guess this filter was removed because its taken care in _getChaptersFromPage.

Reply:
I added it because before I just filtered chapters by `mangaId`.
Using `data.body.page.series` won't produce duplicates. So i removed this filter.